modules/ROOT/pages/managing-users.adoc

```diff
@@ -3,31 +3,85 @@ ifndef::env-site,env-github[]
+
+. From the *Portal members* page, locate the member's name in the list and click *View and assign user groups*.
+. Select one or more user groups and click *Apply*.
+
 == Manage Pending Registration Requests
-Use the *Pending registration requests* page to approve or reject requests for users who registered to access the portal. You can view the user's name, email, registration date, and registration status of the users who register for access. The maximum pending user registration requests is 500.
+Use the *Pending registration requests* page to approve or reject requests for users who registered to access the portal. You can view the user's name, email, registration date, and registration status of the users who register for access. By default, users are assigned to the Members profile. You can add them to the available user groups that can see specific APIs in the portal. The maximum pending user registration requests is 500.
 
 When the registration is approved, the user is removed from the list on the *Pending registration requests* page and added to the list of members on the *Portal members* page. 
 
 If the registration is rejected, the user is removed from the list and can't log in and access the portal.
 
 === Approve or Reject Member Registration
-After a guest user registers for an account using self-registration, approve or reject the request. 
+After a guest user registers for an account using self-registration, approve or reject the requests and assign the user to user groups. All users who self-register are assigned as members. You can assign member users to multiple user groups.
 
 . Click *Pending registration requests*.
 . To find a user, enter the user's name or email in the *Search by name or email* box.
+. To assign the user to a user group:
+.. Click *Members* and select one or more user groups.
+.. Click *Apply*.
 . Click *Approve* or *Reject*.
 
+== Manage User Groups
+Manage user groups using the *User groups* page to create and manage a group of users who can see specific APIs in your portal.   
+
+=== Create User Groups
+Create groups of users who can see specific APIs in the portal.
+
+. From the *User groups* page, click *Create user group*.
+. Enter a name for the group and an optional description.
+. Click *Save*.
+
+
+=== Delete User Groups
+Delete user groups you don't need. Before deleting a user group, remove all users,  APIs, and the visibility for all API instances assigned to the user group. 
+
+To delete a user group, from the *User groups* page, select a user group and click *Delete user group*.
+
+=== Map User Groups to External Identity Providers
+For users of a user group to log in to your portal using SSO, map the user group to an existing external identity provider. To avoid login issues, ensure you use the same group name and identity provider name configured for the identity provider. 
```

Comment:
```suggestion
To let users of a user group log in to your portal using SSO, map the user group to an existing external identity provider. To avoid login issues, use the same group name and identity provider name configured for the identity provider. 
```